src/Model/Photo.php

```diff
@@ -762,7 +762,7 @@ public static function getAlbums(int $uid, bool $update = false): array
 				));
 			} else {
 				// This query doesn't do the count and is much faster
-				$albums = DBA::toArray(DBA::p("SELECT DISTINCT(`album`), '' AS `total`
+				$albums = DBA::toArray(DBA::p("SELECT DISTINCT(`album`), '' AS `total`, ANY_VALUE(`created`) AS `created`
```

Comment:
This changes the behavior of this query by ignoring the lone `DISTINCT()` call. If the goal is to get one line per album, then adding this field breaks the feature. If the created value isn't important, it can be rendered static like the `total` field.
```suggestion
				$albums = DBA::toArray(DBA::p("SELECT DISTINCT(`album`), '' AS `total`, NOW() AS `created`
```

Reply:
We need the value. This is copied from here: https://github.com/friendica/friendica/blob/develop/src/Model/Photo.php#L754C1-L754C127

Reply:
There's a ``COUNT(DISTINCT(`field`))`` call in this query which makes it an aggregate query which warrants the use of `ANY_VALUE`, but it isn't the case in this query where there's no `COUNT`.

Reply:
When the config value `'no_count'` is set, then we don't perform a count, but we use a dummy value for `total`. But the rest of the query must stay the same.

Reply:
Then we need to make this query into an aggregate using ``GROUP BY `album` `` and either `MIN()` or `MAX()` for the `created` date depending on the intention.

Reply:
The difference between these two calls is that one completely omits the `count` since this is really expensive. So we have two different queries.

Reply:
I understand that, but the shortcut we use to omit the count in this query makes it so that we can't have meaningful data in the additional fields. If we need meaningful data in the `created` field we cannot use the `DISTINCT()` shortcut.

Reply:
I changed the query.